heartbeat/monitors/wrappers/monitorstate/tracker.go

```diff
@@ -111,7 +111,7 @@ func (t *Tracker) GetCurrentState(sf stdfields.StdMonitorFields, rc RetryConfig)
 		}
 		var loaderError LoaderError
 		if errors.As(err, &loaderError) && !loaderError.Retry {
-			logp.L().Warnf("could not load last externally recorded state: %w", err)
+			logp.L().Warnf("could not load last externally recorded state: %v", loaderError)
```

Comment:
I guess if we want to use %w, we would need to unwrap `errors.Unwrap`. Not super sure. But %v should called the `Error()` method.